docs/canvas/canvas-getting-started.asciidoc

```diff
@@ -2,31 +2,158 @@
 [[canvas-getting-started]]
 == Getting started with Canvas
 
-Your best bet to getting started with Canvas is to check out one 
-(or all) of the sample data sets that ship with {kib}. 
-
-. Click the {kib} logo in the upper left hand corner of your browser to navigate 
-to the {kib} home page.
-. Click *Load a data set and a Kibana dashboard*. This also loads a 
-Canvas workpad to go with the data set.
-. Pick a data set. Let’s go with the eCommerce one for this example, but while you're 
-here, feel free to click *Add* on all of the available sample data sets.
-. Click the Canvas icon in the left hand navigation menu.
-. Find the *[eCommerce] Revenue Tracking* workpad and click on its name to open it.
+To get up and running with Canvas, use the following tutorial where you'll create a display for monitoring sales at an eCommerce store.
+
+[float]
+=== Before you begin
+
+For this tutorial, you'll use the {kibana-ref}/add-sample-data.html[Sample eCommerce orders data].
+
+[float]
+=== Create and personalize your workpad
+
+Your first step to working with Canvas is to create a workpad. 
+
+. Open *Canvas*.
+
+. Click *Create workpad*.
+
+. To personalize the workpad and page, use the editor on the right.
+
+.. Enter a *Name* for your workpad. For example, `Welcome to Canvas`.
+
+.. Change the page *Background color* to light gray.
+
+[float]
+=== Customize your workpad with images
+
+To customize your workpad to look the way you want, add your own images.
+
+. Click *Add element*, then click *Image*.
++
+The default Elastic logo image appears on your page.
+
+. To replace the Elastic logo with your own image, select the image, then use the editor on the right.
+
+. To move the image, click and drag it to your preferred location.
++
+You'll notice that the image is tagged as an asset, which allows you to reuse the image from *Manage assets*.
+
+[float]
+=== Customize your workpad with text
```

Comment:
This section is more than just customizing a workpad with text.  Steps 1, 2, and 3 are about adding an element and connecting it to a database.  Step 4 is about using the expression language.  Maybe something more along the lines of customizing data.